addons/optics/functions/fnc_handleFired.sqf

```diff
@@ -18,12 +18,12 @@
 
 #include "script_component.hpp"
 
-//IGNORE_PRIVATE_WARNING ["_unit", "_weapon", "_muzzle", "_mode", "_ammo", "_magazine", "_projectile", "_vehicle", "_gunner", "_turret"];
+// IGNORE_PRIVATE_WARNING ["_unit", "_weapon", "_muzzle", "_mode", "_ammo", "_magazine", "_projectile", "_vehicle", "_gunner", "_turret"];
```

Comment:
I'm not sure whether the linter cares about spaces here or not, but have a feeling it will

Reply:
Remove it?

Reply:
should be fine